.github/workflows/scripts/run_tests_python.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/sh
 set -e
 
+bazel clean --expunge
+
 # Python + Bazel
-bazel test --test_output=all --cache_test_results=no //private_set_intersection/python:tests
+bazel test --test_output=all --cache_test_results=no --test_timeout=900 //private_set_intersection/python:tests
```

Comment:
Why is test result caching disabled here? I see this wasn't introduced by this PR, but can you maybe add a comment regarding that, or remove `--cache_test_results=no` if it's not needed any more?